internal/runtime/build/build.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2024 Stefan Prodan
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtimebuild
```

Comment:
This can't be in the runtime package as it creates a cyclic dependency for engine.

Reply:
hm..

```
$ git grep runtimebuild
cmd/timoni/bundle_apply.go:     runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/bundle_apply.go:     runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/bundle_apply.go:     rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, bundleArgs.runtimeFiles...)
cmd/timoni/bundle_build.go:     runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/bundle_build.go:             runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/bundle_build.go:             rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, bundleArgs.runtimeFiles...)
cmd/timoni/bundle_delete.go:    runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/bundle_delete.go:    runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/bundle_delete.go:    rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, bundleArgs.runtimeFiles...)
cmd/timoni/bundle_status.go:    runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/bundle_status.go:    runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/bundle_status.go:    rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, bundleArgs.runtimeFiles...)
cmd/timoni/bundle_vet.go:       runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/bundle_vet.go:       runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/bundle_vet.go:       rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, bundleArgs.runtimeFiles...)
cmd/timoni/runtime_build.go:    runtimebuild "github.com/stefanprodan/timoni/internal/runtime/build"
cmd/timoni/runtime_build.go:    runtimeBuildOpts := runtimebuild.Options{
cmd/timoni/runtime_build.go:    rt, err := runtimebuild.BuildFiles(runtimeBuildOpts, files...)
internal/runtime/build/build.go:package runtimebuild
```

Reply:
I think it's fine if I drop https://github.com/stefanprodan/timoni/pull/366/commits/08c01a865497005dccccf943be11a009f56be0ca, it just looked like a low-hanging fruit, but that code is only used in `cmd` right now anyway.

Reply:
I reverted that commit, please have a look at the overall diff now.